Kconfig

```diff
@@ -9,6 +9,10 @@ mainmenu "RIOT Configuration"
 # For now, get used modules as macros from this file (see kconfig.mk)
 osource "$(KCONFIG_GENERATED_DEPENDENCIES)"
 
+rsource "drivers/Kconfig"
+rsource "sys/Kconfig"
+rsource "pkg/Kconfig"
```

Comment:
Move this together with `sys` and `drivers`

Reply:
Oops. missed that. Now fixed.

Reply:
Please leave `osource "$(APPDIR)/Kconfig"` before the others, like it's currently on master.

Reply:
Done.